garrysmod/lua/includes/modules/cvars.lua

```diff
@@ -18,111 +11,124 @@ module( "cvars" )
 
 local ConVars = {}
 
-function GetConVarCallbacks( name, CreateIfNotFound )
-
-	local Tab = ConVars[ name ]
+--[[---------------------------------------------------------
+	Name: GetConVarCallbacks
+	Desc: Returns a table of the given ConVars callbacks
+-----------------------------------------------------------]]
+function GetConVarCallbacks( name, createIfNotFound )
 
-	if ( CreateIfNotFound && !Tab ) then
-		Tab = {}
-		ConVars[ name ] = Tab
+	local tab = ConVars[ name ]
+	if ( createIfNotFound and !tab ) then
+		tab = {}; ConVars[ name ] = tab

```

Comment:
What's the point of putting these on the same line?

Reply:
just looks shorter and better, i can split it into two lines if you think it's unacceptable

Reply:
Corrected